src/Model/Model_Form_Settings.php

```diff
@@ -184,7 +184,7 @@ public function process_list_view( $form_id ) {
 		$controller->view->list(
 			[
 				'title'       => $this->data->title,
-				'add_new_url' => $add_new_url = add_query_arg( [ 'pid' => 0 ] ),
+				'add_new_url' => $add_new_url = esc_url( add_query_arg( [ 'pid' => 0 ] ) ),
```

Comment:
@jestonihpi this would be better handled at the point the URL is output in the HTML.